crates/notion-core/src/session.rs

```diff
@@ -203,9 +203,19 @@ impl Session {
     /// Fetch, unpack, and install a package matching the input requirements.
     pub fn install_package(&mut self, name: String, version: &VersionSpec) -> Fallible<Version> {
         // fetches and unpacks package
-        let fetched_package = self.fetch_package(name, version)?;
+        let fetched_package = self.fetch_package(name.clone(), version)?;
         let package_version = fetched_package.version();
 
+        // if the package is already installed, don't re-install it
+        if let Fetched::Installed(pkg_version) = fetched_package {
+            let version = pkg_version.version.clone();
+            eprintln!(
```

Comment:
I think we can write this message to `stdout` instead of `stderr`, since it's an informative message about the fact that we didn't need to do anything.

```suggestion
            println!(
```

Reply:
Sounds good to me